packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -386,14 +444,29 @@ export interface BaseSetting {
      */
     caseSensitive?: boolean;
 
-    /** Define additional available dictionaries. */
+    /**
+     * Define additional available dictionaries.
+     *
+     * For example, you can use this to enable both English and Spanish spell-checking at the same
```

Comment:
I think it would be better to use a "project" or "custom" dictionary instead of Spanish. Since this configuration will not enable the Spanish dictionary.

```json
"dictionaryDefinitions": [{ "name": "custom-words", "path": "./custom-words.txt"}],
"dictionaries": ["custom-words"]
```

Reply:
can you offer this as a suggestion so i can click the button "commit suggestion"?